src/erlcloud_s3.erl

```diff
@@ -1014,24 +1014,47 @@ make_link(Expire_time, BucketName, Key, Config) ->
       true  -> lists:flatten([Config#aws_config.s3_scheme, Config#aws_config.s3_host, port_spec(Config), "/", BucketName, "/", Key])
   end.
 
--spec make_get_url(integer(), string(), string()) -> iolist().
-
-make_get_url(Expire_time, BucketName, Key) ->
-    make_get_url(Expire_time, BucketName, Key, default_config()).
+ -spec make_get_url(integer(), string(), string()) -> iolist().
 
+make_get_url(ExpireTime, BucketName, Key) ->
+    make_get_url(ExpireTime, BucketName, Key, default_config()).
+  
 -spec make_get_url(integer(), string(), string(), aws_config()) -> iolist().
 
-make_get_url(Expire_time, BucketName, Key, Config) ->
-    {Sig, Expires} = sign_get(Expire_time, BucketName, erlcloud_http:url_encode_loose(Key), Config),
-    SecurityTokenQS = case Config#aws_config.security_token of
-        undefined -> "";
-        SecurityToken -> "&x-amz-security-token=" ++ erlcloud_http:url_encode(SecurityToken)
-    end,
-    lists:flatten([get_object_url(BucketName, Key, Config),
-     "?AWSAccessKeyId=", erlcloud_http:url_encode(Config#aws_config.access_key_id),
-     "&Signature=", erlcloud_http:url_encode(Sig),
-     "&Expires=", Expires,
-     SecurityTokenQS]).
+make_get_url(ExpireTime, BucketName, Key, Config) ->
+    Region = aws_region_from_host(Config#aws_config.s3_host),
+    Host = "s3-" ++ Region ++ ".amazonaws.com",
+    Path = erlcloud_http:url_encode_loose("/" ++ BucketName ++ "/" ++ Key),
+    EndpointUrl = "https://" ++ Host ++ Path,
```

Comment:
This is not correct because there is no `s3-us-east-1.amazonaws.com` [endpoint](http://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region).  
`Config#aws_config.s3_host` should be used as a `Host` instead.